test/e2e/scenarios/Cluster/cluster_validate_ref.yaml

```diff
@@ -0,0 +1,148 @@
+id: "CLUSTER_VALIDATE_REF"
+description: "In this validate we ref of cluster"
+#marks:
+#  - slow
+#  - blocked
+steps:
+  - id: "USER_CREATE"
+    description: "Create User"
+    create:
+      apiVersion: $CRD_GROUP/$CRD_VERSION
+      kind: User
+      metadata:
+        name: user$RANDOM_SUFFIX
+      spec:
+        name: user$RANDOM_SUFFIX
+        accessString: on +get
+        authenticationMode:
+          type_: password
+          passwords:
+            - key: password
+              name: $SECRET1
+    wait:
+      status:
+        conditions:
+          ACK.ResourceSynced:
+            status: "True"
+            timeout: 180
+  - id: "ACL_CREATE"
+    description: "Create new ACL"
+    create:
+      apiVersion: $CRD_GROUP/$CRD_VERSION
+      kind: ACL
+      metadata:
+        name: acl$RANDOM_SUFFIX
+      spec:
+        name: acl$RANDOM_SUFFIX
+        userNames:
+          - user$RANDOM_SUFFIX
+    wait:
+      status:
+        conditions:
+          ACK.ResourceSynced:
+            status: "True"
+            timeout: 180
+  - id: "PG_INITIAL_CREATE"
+    description: "PG initial create"
+    create:
+      apiVersion: $CRD_GROUP/$CRD_VERSION
+      kind: ParameterGroup
+      metadata:
+        name: pg$RANDOM_SUFFIX
+      spec:
+        description: PG initial create
+        family: memorydb_redis6
+        name: pg$RANDOM_SUFFIX
+    wait:
+      status:
+        conditions:
+          ACK.ResourceSynced:
+            status: "True"
+            timeout: 100
+  - id: "SUBNET_INITIAL_CREATE"
+    description: "Subnet group initial create"
+    create:
+      apiVersion: $CRD_GROUP/$CRD_VERSION
+      kind: SubnetGroup
+      metadata:
+        name: subnet$RANDOM_SUFFIX
+      spec:
+        description: Subnet group create
+        name: subnet$RANDOM_SUFFIX
+        subnetIDs:
+          - $SUBNET1
+          - $SUBNET2
+    wait:
+      status:
+        conditions:
+          ACK.ResourceSynced:
+            status: "True"
+            timeout: 100
+  - id: "CREATE_CLUSTER_VALIDATE_REF"
+    description: "Create a Cluster and validate ref"
+    create:
+      apiVersion: $CRD_GROUP/$CRD_VERSION
+      kind: Cluster
+      metadata:
+        name: cluster$RANDOM_SUFFIX
+      spec:
+        name: cluster$RANDOM_SUFFIX
+        nodeType: db.t4g.medium
+        numShards: 1
+        description: Test create from ACK
+        kmsKeyID: $KMSKEY
+        snsTopicARN: $TOPIC1
+        numReplicasPerShard: 1
+        tlsEnabled: true
+        snapshotWindow: 05:00-06:00
+        snapshotRetentionLimit: 0
+        aclRef:
+          from:
+            name: acl$RANDOM_SUFFIX
+        parameterGroupRef:
+          from:
+            name: pg$RANDOM_SUFFIX
+        subnetGroupRef:
+          from:
+            name: subnet$RANDOM_SUFFIX
+    wait:
+      status:
+        conditions:
+          ACK.ResourceSynced:
+            status: "True"
+            timeout: 3600
```

Comment:
Could you also add a wait for the condition `ACK.ReferencesResolved` - we also expect that to be `true` if `ACK.ResourceSynced = true`

Reply:
Added